plugin/output/s3/s3_test.go

```diff
@@ -407,11 +417,11 @@ func TestStartWithMultiBuckets(t *testing.T) {
 	assert.True(t, size3 > size2)
 }
 
-func newPipeline(t *testing.T, configOutput *Config, objStoreF objStoreFactory) *pipeline.Pipeline {
+func newPipeline(t *testing.T, configOutput *Config, objStoreF objStoreFactory, pipelineCapacity int) *pipeline.Pipeline {
```

Comment:
1. Why do we need to set the pipelineCapacity? It is always 4096
2. Can we return pipeline.Test from this function?